src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketPage.java

```diff
@@ -0,0 +1,51 @@
+package com.cloudbees.jenkins.plugins.bitbucket.client;
```

Comment:
<a href="https://imgflip.com/i/1y9ynw"><img src="https://i.imgflip.com/1y9ynw.jpg" title="made at imgflip.com"/></a>

Reply:
Dam it, why? Add a license file to the repo and get done with it. Hate seeing these license headers in each source files! We do not do in blueocean (most in team don't do it).

Reply:
+1 License file

Reply:
Done.